cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -2756,20 +2756,61 @@ display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
 search = `cs_o365` sourcetype="o365:management:activity" _index_earliest=-61m@m _index_latest=-1m@m Workload IN (OneDrive, SharePoint) Operation IN ("SharingSet", "AccessRequestApproved", "SharingRestored", "AnonymousLinkCreated", "SharingInvitationCreated", "SharingInvitationUpdated", "AddedToSecureLink", "SharingPolicyChanged") NOT TargetUserOrGroupType IN ("Member", "SharePointGroup") \
-| stats values(Operation) AS Operations, values(EventData) AS EventData, values(ItemType) as ItemType, values(ApplicationDisplayName) as ApplicationDisplayName, values(EventSource) as EventSource latest(_time) as LastTime earliest(_time) as FirstTime by SourceFileName, UserId \
-|rename UserId as User \
+| iplocation ClientIP \
+| fillnull value="" Country, Region, City \
+| eval Source = ClientIP \
```

Comment:
Doesn't seems to require "| eval Source = ClientIP".
you can use ClientIP or if you want to rename then use: | rename ClientIP as Source